controllers/storagecluster/cephcluster.go

```diff
@@ -1707,3 +1718,52 @@ func getFullRatio(specifiedRatio *float64) *float64 {
 	defaultRatio := 0.85
 	return &defaultRatio
 }
+
+// updateRookCephOperatorHostNetwork updates the rook-ceph-operator CSV to enable host network
+func updateRookCephOperatorHostNetwork(ctx context.Context, c client.Client, opNamespace string) error {
+	csvList := &opv1a1.ClusterServiceVersionList{}
+	labelSelector := labels.SelectorFromSet(map[string]string{
+		fmt.Sprintf("operators.coreos.com/rook-ceph-operator.%v", opNamespace): "",
```

Comment:
dont we need to use our own labels (Rook provided one) instead of depending on the label set by OLM?

Reply:
@rohan47 Changing the CSV is problematic as when upgrading the new CSV will not include the changes.